protocol/yggdrasil.proto

```diff
@@ -98,3 +102,26 @@ message EventNotification {
     map<string, string> metadata = 3;
 }
 
+// A running-config definition for yggdrasil
+message Config {
+  // Log level
+  string ygg_log_level = 1;
```

Comment:
This can just be `log_level`. There isn't a reason to prefix it with `ygg`, is there?